src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -1757,6 +1770,92 @@ public static void ConsumeLotsOfComments(string valueString, bool insideArray, s
             Assert.Equal(dataUtf8.Length, json.BytesConsumed);
         }
 
+        struct SingleCommentVerifyInfo
+        {
+            public bool startObjectFound;
+            public bool endObjectFound;
+            public bool commentFound;
+        }
+
+        private static void VerifyReadLoop(ref Utf8JsonReader json, string expected, ref SingleCommentVerifyInfo verifyInfo)
+        {
+            while (json.Read())
+            {
+                switch (json.TokenType)
+                {
+                    case JsonTokenType.StartObject:
+                        verifyInfo.startObjectFound = true;
+                        break;
+                    case JsonTokenType.EndObject:
+                        verifyInfo.endObjectFound = true;
+                        break;
+                    case JsonTokenType.Comment:
```

Comment:
I think we only need to check `case JsonTokenType.Comment`, and do nothing for `JsonTokenType.Start/EndObject` (default can continue to be `Assert.True(false)`).